src/PdoInterface.php

```diff
@@ -185,5 +186,5 @@ public function setAttribute($attribute, $value);
      * @return array
      *
      */
-    public static function getAvailableDrivers();
+    //public static function getAvailableDrivers(): array;
```

Comment:
Why commented out?

Reply:
I think I commented it out as static analysis showed it was not used and no tests was falling. I will be adding it back with a corresponding test.